server/models/dataset_helpers/image_metadata.py

```diff
@@ -137,6 +140,8 @@ def _coerceInt(cls, value):
     def _coerceFloat(cls, value):
         try:
             value = float(value)
+            if math.isinf(value) or math.isnan(value):
+                raise ValueError
```

Comment:
Typically, I would expect to see `raise ValueError()`, but it looks like this form will automatically instantiate the passed class: https://docs.python.org/2/reference/simple_stmts.html#raise , and we use this syntax in other places too.